### PR TITLE
early pricing initial commit

### PR DESCRIPTION
**Pull request by paulatulis**
Pricing Cards: 
- Four plans
- Filled in CTA button on Professional and set box shadow to emphasize
- Added additional user info under price
- Updated bullets and font size
- Still need to have unique paths for the Request Early Access submit buttons. Using the enterprise form as a placeholder. @ksatx are you able to provide?

Logos: 
- Updated title to 'Leading Organizations Trust Stoplight with their APIs'

Grid: 
- Added tool tip copy
- Need to adjust some small css details to get things centered on different screen sizes
- Need paths for each CTA button at the bottom of the grid

Action Bar: 
- No changes

FAQs:
- Undid click to reveal more functionality
- The huge difference in the lengths of the answers makes this look super awkward with spacing when they are wrapped side-by-side, so I left it unwrapped. Open to suggestions.

Screen Shot 2020-05-13 at 2 26 14 PM (2)

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit c10cd576f5479e4b4eb8d28d51bb289a48df9cad

https://deploy-preview-1452--stoplightio.netlify.app

**Comment by mnaumanali94** [comment]
I think it would be nice if we could:
 1. Change unlimited in tiles to https://fontawesome.com/icons/infinity and remove the checkmark icon. 
2. Change the icon beside `everything in plan` to https://fontawesome.com/icons/arrow-left?style=solid

Analog: https://github.com/pricing

**Comment by mnaumanali94** [comment]
Add a `Compare Plans` link right after the plans which takes the user to the feature breakdown. 

Analog: https://www.figma.com/pricing/

**Comment by mnaumanali94** [comment]
I updated the FAQs at https://docs.google.com/spreadsheets/d/1dSxt8OJvzbVWSEcwxNQfVo-sk8eG_RUUZaZY4BHmIEs/edit#gid=1191199743 

Tried to make the answers a little more equal in length. Hope this helps.

**Comment by mnaumanali94** [comment]
@smf434 Some comments you can go through.

**Comment by mnaumanali94** [comment]
Screenshot 2020-05-14 at 1 19 17 AM

We can potentially organize the top links as:  Products - Open Source - Enterprise - Pricing - Resources - Blog - About

Just a thought.

**Comment by ksatx** [comment]
@paulatulis 


**Free Card Form**

`
<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
<script>
  hbspt.forms.create({
	portalId: "4892910",
	formId: "91b8fe5d-eab8-4c67-b0cc-f1d56ebecef4",
	sfdcCampaignId: "7013i000000kofQAAQ"
});
</script>`

**Starter Form**

`
<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
<script>
  hbspt.forms.create({
	portalId: "4892910",
	formId: "23988ae4-a448-4b62-b34e-00f10dbb50c8",
	sfdcCampaignId: "7013i000000kofVAAQ"
});
</script>`

**Professional Form**

`
<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
<script>
  hbspt.forms.create({
	portalId: "4892910",
	formId: "3f2257fa-1dd9-4670-b1a7-9cd0d8e50ce2",
	sfdcCampaignId: "7013i000000kofaAAA"
});
</script>`

**Enterprise Form**

`
<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
<script>
  hbspt.forms.create({
	portalId: "4892910",
	formId: "00687bee-29d9-4a25-8abd-be15d11efe9e",
	sfdcCampaignId: "7013i000000kofWAAQ"
});
</script>`

**Comment by ksatx** [comment]
@paulatulis let me know when you've committed those forms to this branch, I'm curious to see what the Hubspot generated "Thank You" message looks like before any extra styling.

**Comment by paulatulis** [comment]
> Screenshot 2020-05-14 at 1 19 17 AM
> 
> We can potentially organize the top links as: Products - Open Source - Enterprise - Pricing - Resources - Blog - About
> 
> Just a thought.

The way the header is configured has anything with a dropdown separate from the pure buttons, So this will take more work to reconfigure. I think we should re-address it when we do the main website changes, though!

**Comment by mnaumanali94** [comment]
The concern was pricing being too far off. This is good.

**Comment by paulatulis** [comment]
> @paulatulis
> 
> **Free Card Form**
> 
> `
> 
> <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script> <script> hbspt.forms.create({ portalId: "4892910", formId: "91b8fe5d-eab8-4c67-b0cc-f1d56ebecef4", sfdcCampaignId: "7013i000000kofQAAQ" }); </script>`
> **Starter Form**
> 
> `
> 
> <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script> <script> hbspt.forms.create({ portalId: "4892910", formId: "23988ae4-a448-4b62-b34e-00f10dbb50c8", sfdcCampaignId: "7013i000000kofVAAQ" }); </script>`
> **Professional Form**
> 
> `
> 
> <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script> <script> hbspt.forms.create({ portalId: "4892910", formId: "3f2257fa-1dd9-4670-b1a7-9cd0d8e50ce2", sfdcCampaignId: "7013i000000kofaAAA" }); </script>`
> **Enterprise Form**
> 
> `
> 
> <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script> <script> hbspt.forms.create({ portalId: "4892910", formId: "00687bee-29d9-4a25-8abd-be15d11efe9e", sfdcCampaignId: "7013i000000kofWAAQ" }); </script>`

Not great.... this might be difficult to correct in a short amount of time. Brainstorming what we could do instead.

Screen Shot 2020-05-13 at 4 40 39 PM
Screen Shot 2020-05-13 at 4 40 49 PM

**Comment by paulatulis** [comment]
> @paulatulis let me know when you've committed those forms to this branch, I'm curious to see what the Hubspot generated "Thank You" message looks like before any extra styling.

Should be in the next preview. Looks rough right now, but I'm sure there's a way to fix it? This will be my next area of focus

**Comment by ksatx** [comment]
> > @paulatulis let me know when you've committed those forms to this branch, I'm curious to see what the Hubspot generated "Thank You" message looks like before any extra styling.
> 
> Should be in the next preview. Looks rough right now, but I'm sure there's a way to fix it? This will be my next area of focus

I changed the the "Starter" form to redirect to the homepage on submission, I'll take a look at the preview when it's ready. Hubspot requires an action on submit, so maybe we can throw up a quick thank you page? 
image

**Comment by paulatulis** [comment]
> I updated the FAQs at https://docs.google.com/spreadsheets/d/1dSxt8OJvzbVWSEcwxNQfVo-sk8eG_RUUZaZY4BHmIEs/edit#gid=1191199743
> 
> Tried to make the answers a little more equal in length. Hope this helps.

Thank you! This helps a lot.

**Comment by paulatulis** [comment]
> Add a `Compare Plans` link right after the plans which takes the user to the feature breakdown.
> 
> Analog: https://www.figma.com/pricing/

Added. Please let me know if this works for you.

**Comment by mnaumanali94** [comment]
> > Add a `Compare Plans` link right after the plans which takes the user to the feature breakdown.
> > Analog: https://www.figma.com/pricing/
> 
> Added. Please let me know if this works for you.
Looks good. Can we call it `Compare Plans` and make the button just an outline instead of filled. Also if we could lessen the padding a little (If easy) that would be nice.

**Comment by paulatulis** [comment]
> > > Add a `Compare Plans` link right after the plans which takes the user to the feature breakdown.
> > > Analog: https://www.figma.com/pricing/
> > 
> > 
> > Added. Please let me know if this works for you.
> 
> Looks good. Can we call it `Compare Plans` and make the button just an outline instead of filled. Also if we could lessen the padding a little (If easy) that would be nice.

lmk what you think of this iteration

**Comment by mnaumanali94** [comment]
Looks good.

**Comment by paulatulis** [comment]
Top Cards: 
- 'Request Early Access' buttons redirect to home page instead of producing Thank You message that makes css do weird things- this will change again
- Infinity icon for 'Unlimited Users/Projects' bullets
- Arrow icon for 'Everything from X' bullets
- Compare plans anchor link goes to grid

Grid: 
- CTA buttons at the bottom anchor link back up to the 'Request Early Access' buttons


FAQs
-Flex wrap into columns 
- Consolidated question about discounts for OS and nonprofits into one

TO DO:
- fix responsive layout across the board

File: netlify/pages/pricing.yaml [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+            href: '/demo/'
+          input:
+            type: email
+            placeholder: Your work email...
+          formId: '3f2257fa-1dd9-4670-b1a7-9cd0d8e50ce2'
+          portal: '4892910'
+          sfdcCampaignId: '7013i000000kofaAAA'
+  - title: Enterprise
+    titleColor: orange
+    summary: 'Security, best-in-class support and flexible deployment for enterprises'
```
Comment (ksatx) [inline netlify/pages/pricing.yaml:111]: Add Oxford comma - "Security, best-in-class support, and flexible deployment for enterprises"

**Comment by ksatx** [comment]
@smf434  The way we phrase the per-user pricing is odd to me. We characterize it as "for growing API teams" but the baseline pricing is essentially for an individual plan. What about something like below?

pricing wording

@paulatulis At the top of the Enterprise Card, "Contact Us" should be the same size and weight as "$0" "$99" "$399"
On the bottom, the CTA on the Enterprise Form feels a little misleading, like a chat window will pop up. "Contact Us" seems more appropriate. 

> Much better! My only concern is that it will then direct them away from the pricing page. Is that going to be a blocking objection we get?

We can get Scott's input on the redirect in the growth sync. Styling a modal feels like a big time investment for a temporary page.

File: netlify/pages/pricing.yaml (hunk 1 above)
Comment (paulatulis) [reply netlify/pages/pricing.yaml:111]:
```suggestion
    summary: 'Security, best-in-class support ,and flexible deployment for enterprises'
```

**Comment by paulatulis** [comment]
Latest updates: 

Top Cards:
- corrected css for HubSpot submit thank you message and changed the Starter plan back to the thank-you message option in the form settings
- changed add on pricing line 

I think the plus icon should go to the left of the line. When it's above it, it stretches the top portion of the card out too much and the others to the side look even more bare 
Screen Shot 2020-05-14 at 11 50 48 AM

Still working on better layout for medium sized screens on the grid. Please let me know what else I am missing.

**Comment by mnaumanali94** [comment]
1. I would increase the size of the user pricing a little. Thoughts?

2. Screenshot 2020-05-14 at 11 00 47 PM

Could we potentially have a single input here for the email above these CTAs?

**Comment by paulatulis** [comment]
> 1. I would increase the size of the user pricing a little. Thoughts?
> 2. Screenshot 2020-05-14 at 11 00 47 PM
> 
> Could we potentially have a single input here for the email above these CTAs?

What do you mean by single input for email? I think it needs to be different since they are all separate forms, but maybe I'm misunderstanding what you mean.

**Comment by mnaumanali94** [comment]
Like just input the work email on a single field above these buttons, and whichever button they press it ties the input to that particular form in Hubspot. If that is easily doable. Otherwise, it's fine to have separate inputs. 

Oh, I meant increasing the size of this: 
Screenshot 2020-05-14 at 11 08 06 PM

**Comment by ksatx** [comment]
> Like just input the work email on a single field above these buttons, and whichever button they press it ties the input to that particular form in Hubspot. If that is easily doable. Otherwise, it's fine to have separate inputs.

I don't think that's do-able, at least from a Hubspot perspective.

**Comment by mnaumanali94** [comment]
> > Like just input the work email on a single field above these buttons, and whichever button they press it ties the input to that particular form in Hubspot. If that is easily doable. Otherwise, it's fine to have separate inputs.
> 
> I don't think that's do-able, at least from a Hubspot perspective.

Sounds good.

File: netlify/pages/pricing.yaml [hunk 2]
```diff
@@ -0,0 +1,619 @@
...
+            href: '/demo/'
+          input:
+            type: email
+            placeholder: Your work email...
+          formId: '3f2257fa-1dd9-4670-b1a7-9cd0d8e50ce2'
+          portal: '4892910'
+          sfdcCampaignId: '7013i000000kofaAAA'
+  - title: Enterprise
+    titleColor: orange
+    summary: 'Security, best-in-class support ,and flexible deployment for enterprises'
```
Comment (mnaumanali94) [inline netlify/pages/pricing.yaml:111]:
```suggestion
    summary: 'Security, best-in-class support, and flexible deployment for enterprises'
```

File: netlify/pages/pricing.yaml [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+      href: '/demo/'
+      outlined: true
+    price:
+      # title: Contact Us
+      # button:
+      color: purple
+      title: Contact Us
+      href: '/demo/'
+    features:
+      - name: Unlimited Users
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:124]:
```suggestion
      - name: No User Cap
```

Comment (paulatulis) [reply netlify/pages/pricing.yaml]: @mnaumanali94 thank you for catching this! just did it here. If you see small things like this, feel free to leave suggestions and then commit them as you like.

File: netlify/pages/pricing.yaml [hunk 4]
```diff
@@ -0,0 +1,619 @@
...
+      - name: Customer Success Manager
+      - name: Custom Terms
+      - name: Vendor Security Forms
+    ctas:
+      - title: Request Early Access
+        type: 'submit'
+        submit:
+          button:
+            color: orange
+            title: Chat With Us
```
Comment (ksatx) [inline netlify/pages/pricing.yaml:138]: Contact Us

Comment (ksatx) [inline netlify/pages/pricing.yaml:138]:
```suggestion
            title: Contact Us
```

**Pull request merged**